firebase-installations/src/main/java/com/google/firebase/installations/Utils.java

```diff
@@ -43,7 +48,12 @@ public boolean isAuthTokenExpired(PersistedInstallationEntry entry) {
 
   /** Returns current time in seconds. */
   public long currentTimeInSecs() {
-    return TimeUnit.MILLISECONDS.toSeconds(System.currentTimeMillis());
+    return TimeUnit.MILLISECONDS.toSeconds(currentTimeInMillis());
+  }
+
+  /** Returns current time in milliseconds. */
+  public long currentTimeInMillis() {
```

Comment:
static?

Reply:
Mockito doesn't allow to mock static methods. As a result methods in this class are not static.

Reply:
Please add a respective comment (not Javadoc) to the code.